### PR TITLE
Added bci studies p300 files and readme

### PR DESCRIPTION
**Pull request by yegorkt25**

### CODE REVIEW

File: bci-study-p300/experiment_recorder/p300_2/p300_2.cfg [hunk 1]
```diff
@@ -0,0 +1,4 @@
+p300_2
+
+a = [(15, 0.1), (15, 0.2), (15, 0.3), (15, 0.1), (15, 0.2), (15, 0.3), (15, 0.1), (15, 0.2), (15, 0.3), (15, 0.1), (15, 0.2), (15, 0.3), (15, 0.1), (15, 0.2), (15, 0.3)]
```
Comment (MichaelAntropov) [inline bci-study-p300/experiment_recorder/p300_2/p300_2.cfg:3]: What does `a` mean?

Comment (yegorkt25) [reply bci-study-p300/experiment_recorder/p300_2/p300_2.cfg]: fixed

**Review by MichaelAntropov — COMMENTED** [review]
Start commits with upper letter

File: bci-p300-sensors-reposition/README.md [hunk 2]
```diff
@@ -1,3 +1,50 @@
...
+['fp1','fp2','c3','c4','p7','p8','o1','o2','f7','f8','f3','f4','t7','t8','p3','p4']
+```
+
+It is important since we will be moving sensors and will have to 'relabel' channels.
+
+#### Relocating sensors:
+
+The new location of the sensors will be moved as follows:
+
+| Original |    | New | Ch No. |   |
```
Comment (MichaelAntropov) [inline bci-p300-sensors-reposition/README.md:27]: Fix table :)

**Review by MichaelAntropov — APPROVED** [review]
LGTM

**Pull request merged**